src/testdir/test_quickfix.vim

```diff
@@ -2038,6 +2038,16 @@ func QfAutoCmdHandler(loc, cmd)
 endfunc
 
 func Test_Autocmd()
+
+  set tags=Xtags
+  call writefile(["!_TAG_FILE_ENCODING\tutf-8\t//",
+        \ "one\tXfile1\t/^one/;\"\tf\tfile:\tsignature:(void)"],
+        \ 'Xtags')
+  new Xfile1
+  call setline(1, ['empty', 'one()', 'empty'])
+  write
+  bwipe!
```

Comment:
The creation of Xfile1 can be simplified by using
call writefile(['empty', 'one()', 'empty'], 'Xfile1')